packages/api/src/graphql/query.public.ts

```diff
@@ -70,12 +70,17 @@ export const GraphQLPublicQuery = new GraphQLObjectType<undefined, Context>({
       type: GraphQLPeer,
       args: {id: {type: GraphQLID}, slug: {type: GraphQLSlug}},
       description: 'This query takes either the ID or the slug and returns the peer profile.',
-      resolve(root, {id, slug}, {loaders}) {
+      async resolve(root, {id, slug}, {loaders}) {
         if ((id == null && slug == null) || (id != null && slug != null)) {
           throw new UserInputError('You must provide either `id` or `slug`.')
         }
 
-        return id ? loaders.peer.load(id) : loaders.peerBySlug.load(slug)
+        const toReturn = id ? await loaders.peer.load(id) : await loaders.peerBySlug.load(slug)
```

Comment:
I think it makes more sense to give that variable a name of what it is, than what it's used for. What do you think? :)